spotty/spotty/Utils/Extensions/UICollectionView+configure.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  UICollectionView+configure.swift
+//  spotty
+//
+//  Created by Ivaylo Kalaydzhiev on 20.06.22.
+//
+
+import UIKit
+
+extension UICollectionView {
+    
+    func configureReuseableCell<T: ReuseableCell, H: BusinessModel>(_ cellType: T.Type,
+                                                                    modelType: H.Type,
+                                                                    item: AnyHashable,
+                                                                    indexPath: IndexPath) -> T {
+        guard let cell = self.dequeueReusableCell(
+            withReuseIdentifier: cellType.reuseIdentifier,
+            for: indexPath) as? T,
+              let model = item as? H
+        else { fatalError("Developer Error!") }
```

Comment:
In the development process we are ok to throw fatal errors in our guard statements, but when we release we want to return like just an insertion to UICollectionVeiwCell() so out app will continue running and just won't display the cell we expect.

Reply:
Done

Reply:
Let's revert to the state with fatalError and in future we will introduce a default cell for such a cases.

Reply:
done